src/Database/Schema/MysqlSchema.php

```diff
@@ -393,15 +404,20 @@ public function columnSql(TableSchema $schema, string $name): string
             $out .= '(' . $data['length'] . ')';
         }
 
-        $hasPrecision = [TableSchema::TYPE_FLOAT, TableSchema::TYPE_DECIMAL];
-        if (in_array($data['type'], $hasPrecision, true) && isset($data['length'])) {
+        $hasLengthAndPrecision = [TableSchema::TYPE_FLOAT, TableSchema::TYPE_DECIMAL];
+        if (in_array($data['type'], $hasLengthAndPrecision, true) && isset($data['length'])) {
             if (isset($data['precision'])) {
                 $out .= '(' . (int)$data['length'] . ',' . (int)$data['precision'] . ')';
             } else {
                 $out .= '(' . (int)$data['length'] . ')';
             }
         }
 
+        $hasPrecision = [TableSchema::TYPE_DATETIME_FRACTIONAL, TableSchema::TYPE_TIMESTAMP_FRACTIONAL];
```

Comment:
Rather `$precisionTypes` right?

Reply:
I was just following the existing naming.  I can update it.